backend/experiments/StdLibExperimental/Libs/Experiments.fs

```diff
@@ -110,20 +110,25 @@ let fns : List<BuiltInFn> =
         function
         | _, _, [ DString code ] ->
           uply {
-            let canvas = Parser.CanvasV2.parse code
+            try
+              let canvas = Parser.CanvasV2.parse code
```

Comment:
Wrapping only the parsing line in a try/with block wasn't effective. I wasn't receiving the same error messages as when I wrapped the entire function. Was I doing it wrong?
```
let canvas =
              try
                Parser.CanvasV2.parse code
              with
              | e ->
                Exception.raiseInternal "Error parsing code" [ "error", e.Message ]
              ...
```